GridList/GridList.jsx

```diff
@@ -0,0 +1,153 @@
+import { h } from "preact";
+import MaterialComponent from "../MaterialComponent";
+import Icon from "../Icon";
+
+const VALIDATION_VALUES_BY_KEY = {
+  "with-icon-align": ["start", "end"],
+  "tile-gutter": [1],
+  "tile-aspect-ratio": ["1x1", "16x9", "2x3", "3x2", "4x3", "3x4"]
+};
+
+const isValidValue = (validationValues, testValue) => {
+  return (
+    validationValues &&
+    validationValues.findIndex(val => val === testValue) >= 0
+  );
+};
+
+const notEmptyString = val => val !== "";
```

Comment:
Why are these constants / functions not in the ```GridList``` class?

Reply:
No practical reason. Purely code style. They can be moved in if preferred.

Reply:
I moved them into the class. Is this what you were envisioning?

Reply:
Yes, thanks.